src/planscape/collaboration/utils.py

```diff
@@ -0,0 +1,38 @@
+from collaboration.models import Collaborator, Role
+from planning.models import PlanningArea
+
+
+def get_collaborator(user_id, planning_area_id):
```

Comment:
I'm wondering if it might be cleaner to have a function that accepts a role, unless you think there's value in returning a record, then testing for a role on it as separate steps?